core/src/com/mygdx/insanerpg/model/unit/BattleMage.java

```diff
@@ -0,0 +1,9 @@
+package com.mygdx.insanerpg.model.unit;
+
+class BattleMage extends Player{
+
+    public BattleMage() {
+        super(100, 200, "Ye Qiu", 1, 1, 10, 1.5, 1.5, true,
```

Comment:
Make it take in a name, and set that as the player name.

Reply:
changed "ye qiu" to variable T_T